operator/gitops/argocd/pipeline-service/metrics-exporter/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ resources:
 images:
   - name: quay.io/redhat-appstudio/pipeline-service-exporter
     newName: quay.io/redhat-appstudio/pipeline-service-exporter
-    newTag: 02edcf3b3a394611ae4c514ec689801bcf811836
+    newTag: c594f5d4b8e15eedd6df22082ae966f9ae24f713
```

Comment:
this tag is present up at https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags